frontend/simple/components/MembersCircle.vue

```diff
@@ -1,90 +1,197 @@
 <template>
-  <div>
-    <ul class='circle-container'>
-      <li v-for="(member, username) in members">
-        <img :src='member.attrs.picture' :style="member.style">
+  <div class="gi-memberCircle-container">
+    <svg viewBox="0 0 320 320" version="1.1" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" class="gi-svg">
+      <circle cx="160" cy="160" r="158" class="gi-svg-circle"/>
+    </svg>
+
+    <ul class="gi-circle-list">
+      <li v-for="(member, username) in membersWithStyle.slice(0, 6)" class="gi-item">
+        <div class="gi-item-box"
+          :style="member.style"
+          v-on:click="toggleMemberActive(member)"
+          v-on:mouseenter="toggleMemberActive(member, true)"
+          v-on:mouseleave="toggleMemberActive(member, false)"
+          >
+          <img class="gi-item-img"
+            :src='member.attributes.picture'
+            :alt="`${member.attributes.name}'s avatar`">
+        </div>
+      </li>
+
+      <li v-if="members.length > maxMembers" class="gi-item">
+        <div class="gi-item-box"
+          :style="membersWithStyle[maxMembers].style"
+          v-on:click="toggleMemberActive('other')"
+          v-on:mouseenter="toggleMemberActive('other', true)"
+          v-on:mouseleave="toggleMemberActive('other', false)"
+          >
+          <div class="gi-item-others is-flex">
+            <i class="fa fa-plus is-size-4"></i>
+          </div>
+        </div>
```

Comment:
FYI, I wasn't able to test this locally because the invite/voting system isn't finished. (#343) /cc @hubudibu

Reply:
Really? I've developed it all locally. Just create 2 members tt1 and tt2. tt1 invites tt2 to his group and tt2 go to inbox and the invitation is there.

Reply:
This is in reference to more than 3 members, and this code only kicks in after 6 or more (or more than 6), so that's the part that I couldn't test.